<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>290</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>640</width>
    <height>290</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>649</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QFrame" name="frame_3">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>5</y>
     <width>631</width>
     <height>231</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color : qlineargradient(spread:pad, x1:0.00568182, y1:0, x2:1, y2:0, stop:0.0511364 rgba(37, 105, 209, 255), stop:1 rgba(19, 38, 69, 200))</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QFrame" name="frame_5">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>10</y>
      <width>391</width>
      <height>111</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color : none;
image: url(resources/images/simple white.png);
border: none;</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>611</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color : none;
font: 500 14pt &quot;Cascadia Mono&quot;;
color: white;
</string>
    </property>
    <property name="text">
     <string>Syst√®me de Reconnaissance des Gestes</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QFrame" name="frame_6">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>631</width>
      <height>231</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(23, 49, 90,0.2);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QSplitter" name="splitter">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>150</y>
       <width>81</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color : none;</string>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="handleWidth">
      <number>1</number>
     </property>
     <widget class="QLabel" name="label_8">
      <property name="styleSheet">
       <string notr="true">background-color : none;
font: 200 10pt &quot;Cascadia Mono&quot;;
color: white;
</string>
      </property>
      <property name="text">
       <string>Version :</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="styleSheet">
       <string notr="true">background-color : none;
font: 200 10pt &quot;Cascadia Mono&quot;;
color: white;
</string>
      </property>
      <property name="text">
       <string>Platform</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="styleSheet">
       <string notr="true">background-color : none;
font: 200 10pt &quot;Cascadia Mono&quot;;
color: white;
</string>
      </property>
      <property name="text">
       <string>Licence:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
     </widget>
    </widget>
    <widget class="QSplitter" name="splitter_2">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>150</y>
       <width>241</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: none;</string>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="opaqueResize">
      <bool>true</bool>
     </property>
     <property name="handleWidth">
      <number>1</number>
     </property>
     <widget class="QLabel" name="labelVersion">
      <property name="styleSheet">
       <string notr="true">background-color : none;
font: 200 10pt &quot;Cascadia Mono&quot;;
color: white;
</string>
      </property>
      <property name="text">
       <string>1.0v</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="labelClient">
      <property name="styleSheet">
       <string notr="true">background-color : none;
font: 200 10pt &quot;Cascadia Mono&quot;;
color: white;
</string>
      </property>
      <property name="text">
       <string>Windows / Linux</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="labelLicence">
      <property name="styleSheet">
       <string notr="true">background-color : none;
font: 200 10pt &quot;Cascadia Mono&quot;;
color: white;
</string>
      </property>
      <property name="text">
       <string>Opensource</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </widget>
   </widget>
   <zorder>frame_6</zorder>
   <zorder>frame_5</zorder>
   <zorder>label</zorder>
  </widget>
  <widget class="QFrame" name="progressFrame">
   <property name="geometry">
    <rect>
     <x>6</x>
     <y>247</y>
     <width>630</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(23, 49, 90);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>611</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color : none;
font: 200 10pt &quot;Cascadia Mono&quot;;
color: white;
</string>
    </property>
    <property name="text">
     <string>Chargements ...</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>-5</x>
     <y>-5</y>
     <width>2</width>
     <height>2</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2"/>
  </widget>
  <zorder>layoutWidget</zorder>
  <zorder>frame_3</zorder>
  <zorder>progressFrame</zorder>
 </widget>
 <resources>
  <include location="../images/img_resources.qrc"/>
  <include location="../images/images.qrc"/>
  <include location="../images/pictures.qrc"/>
 </resources>
 <connections/>
</ui>
